canonicalwebteam/blog/django/views.py

```diff
@@ -62,8 +62,9 @@ def upcoming(request):
 
 
 def author(request, username):
+    page_param = request.GET.get("page", default="1")
```

Comment:
Will this crash when inserting `abc` for page?

Reply:
Opened an issue for it: https://github.com/canonical-web-and-design/canonicalwebteam.blog/issues/88